app/src/db/models/tables/objectModel.js

```diff
@@ -153,6 +153,9 @@ class ObjectModel extends Timestamps(Model) {
             });
         }
       },
+      findPath(query, value) {
```

Comment:
wonder if this should be named `filterPath` like other modifiers.

Reply:
filterPath preserves the existing ilike searching behavior (ref Line 68) mostly used by the search endpoints; findPath is a bit more precise as it does an exact match instead.